.github/workflows/trivy.yml

```diff
@@ -0,0 +1,29 @@
+
+name: trivy
+
+on:
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trivy:
+    name: trivy
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
+          scan-type: 'fs'
+          scan-ref: 'ops/dev/'
```

Comment:
We'll need this to scan more than just the `dev` folder. Can we expand the scope to the entirety of `ops`?

Reply:
Done